docs/src/main/sphinx/release/release-418.md

```diff
@@ -0,0 +1,61 @@
+# Release 418 (17 May 2023)
+
+## General
+
+* Add support for `EXECUTE IMMEDIATE`. ({issue}`17341`)
+* Improve performance of joins on partitioned columns. ({issue}`14493`)
```

Comment:
What's a "partitioned column". Is this a reference to a Hive or Iceberg "partition column"? If so, it should go in the corresponding section(s). 

cc @raunaqmorarka

Reply:
Looks like it does reference partitioned columns from various connectors - I'll move it to Hive/Iceberg/Hudi/Delta, but are there any others?

Reply:
Yes, it refers to a partition column in data lake connectors because that's the most straightforward case. The change is not in any connector though, it's in the engine join operator. So any connector which can produce RLE blocks for the engine can benefit, but that's a low level detail.

Reply:
Yeah, I think we just want to frame it as where users will see this improvement. So data lake sections it is.